### ISSUE TITLE
"Build image..." command should provide more files

### ISSUE THREAD

**Issue by yfdyh000** [microsoft/vscode-docker#3163]
Like the "Dockerfile.arm". It only lists files that fully match the “Dockerfile”, anywhere in the working directory. I hope it provides more fuzzy match to the file name.

Alternative, I can select this command from the context menu of the explorer of VSCode.

**Comment by bwateratmsft** [comment]
This is possible using the "files.associations" setting in user settings or `.vscode/settings.json` within the workspace directory. Here's an example:

```json
    "files.associations": {
        "ZooDockerfile": "dockerfile"
    }
```

![image](https://user-images.githubusercontent.com/36966225/130449306-fff235e4-595e-4f1e-bee5-e8349b7b571f.png)

**Comment by yfdyh000** [comment]
This menu works for me, but the issue is the command.

Steps to reproduce:
1. Press F1 key.
2. Type and use the `>Docker Images: Build Images...` command.
3. It lists the any "Dockerfile" file, but not have the 'Dockerfile.*' files and more.
![image](https://user-images.githubusercontent.com/1769875/130533420-e046c6d7-600d-4769-99be-59ce36f8afea.png)

**Comment by bwateratmsft** [comment]
`files.associations` works for that too:

![image](https://user-images.githubusercontent.com/36966225/130617236-c4b49c08-f21d-405c-a8c8-7984641b47c9.png)

There is a gap though; the language definition includes "Dockerfile", "*.dockerfile", and "Dockerfile.*" but the glob patterns for the quick pick include only the first two. I'll get that last one added.

**Comment by bwateratmsft** [comment]
The fix for this has been released in Docker extension 1.17.0.
